### DISCUSSION TITLE
Horizontal distance of the implement

### DISCUSSION THREAD

**Discussion by franklinleites** [Courseplay/Courseplay_FS22#3079] [Q&A]
Hello, I come to ask for help with my doubts about CP. It concerns problems with the plow, where the mods I am using are causing issues in the field. It could be easily adjusted by the horizontal distance of the implement, but this function becomes automatic when the plow is attached and does not allow any changes. I have tested it with both mods and the original game equipment. I would appreciate assistance with this matter.
![2024-01-23 (1)](https://github.com/Courseplay/Courseplay_FS22/assets/157546087/fd9605b1-a78c-4696-90e6-f7db87cffdcb)
![2024-01-23 (2)](https://github.com/Courseplay/Courseplay_FS22/assets/157546087/b499c2ae-5964-4b53-9ad1-e660328fd010)
![2024-01-23 (3)](https://github.com/Courseplay/Courseplay_FS22/assets/157546087/2501f78f-8f3f-43e9-8dd9-7a13eecb7427)
![2024-01-23](https://github.com/Courseplay/Courseplay_FS22/assets/157546087/8ba4942d-5aef-47f1-af4d-a16af414a556)

**Comment by Tensuko** [comment]
As this is not a plow and wrong build or converted, it wont work and cant the correct offset.
Use an actual plow.

**Reply by Tensuko** [reply]
Use this one if you dont want normal plows, because this one is correctly build.
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=226965&title=fs2022

**Reply by franklinleites** [reply]
Hello! These plows are directly downloaded from the mod hub. My question is whether it's possible to set the vehicle offset in a regular manner so that I can adjust it according to my needs. This is because in other implements such as planters and seeders, the mod is manually set for adjustment. So, my question is whether it's possible to manually set the implement offset when I attach any type of plow, whether it's original to the game or a mod.

**Reply by Tensuko** [reply]
Create them as a plow in the GE.
You can try to put it in your custom vehicles file, but if something isnt working right, it is due to the mod.
We will not do anything here.

**Reply by Tensuko** [reply]
https://github.com/Courseplay/Courseplay_FS22/wiki/The-VehicleConfigurations.xml-file.

**Reply by franklinleites** [reply]
Okay, thank you for your patience and attention. I opted to use a plow that does not have this flaw.
